eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/ZmqWatcher.scala

```diff
@@ -100,9 +100,28 @@ class ZmqWatcher(chainHash: ByteVector32, blockCount: AtomicLong, client: Extend
           blockCount.set(count)
           context.system.eventStream.publish(CurrentBlockCount(count))
       }
-      client.rpcClient.invoke("getbalance").collect {
-        // We track our balance in mBTC because a rounding issue in BTC would be too impactful.
-        case JDecimal(balance) => Metrics.BitcoinBalance.withoutTags().update(balance.doubleValue * 1000)
+      // TODO: move to a separate actor that listens to CurrentBlockCount
```

Comment:
nit: would it be better to move this to a Github Issue such that newer contributors could take a stab at this?

Reply:
I have plans for this one, it will not be a good first issue: this will be the basis of our fee bumping / utxo manager component, I'll handle this one ;)

Reply:
But we do have a "good first issue" tag and some issues waiting to be picked up by new contributors! Don't hesitate to ask questions directly on these issues if one looks interesting to you.